### PR TITLE
Update common file to 1.6 branch; update CODEOWNERS

### PR DESCRIPTION
**Pull request by jacob-delgado**
Similar to https://github.com/istio/api/pull/1264 and https://github.com/istio/api/pull/1256

### CODE REVIEW

**Comment by jacob-delgado** [comment]
There is a `Makefile.common.mk` that is not altered (either in this or the one in 1.5). I assume that is correct since it was not merged in for 1.5

File: mcp/Readme.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 This folder contains the proto buffers for the Mesh Configuration
 Protocol (MCP). MCP is based on
-[XDS](https://github.com/envoyproxy/data-plane-api/blob/master/xds_protocol.rst#streaming-grpc-subscriptions)
+[XDS](https://github.com/envoyproxy/data-plane-api/blob/release-1.6/xds_protocol.rst#streaming-grpc-subscriptions)
```
Comment (brian-avery) [inline mcp/Readme.md:7]: this link isn't valid

Comment (brian-avery) [reply mcp/Readme.md:7]: it should be left at master. Envoy repo :)

Comment (jacob-delgado) [reply mcp/Readme.md]: fixed

**Pull request merged**